src/include/sof/audio/codec_adapter/codec/generic.h

```diff
@@ -44,8 +56,10 @@ struct codec_config {
 };
 
 struct codec_data {
+	enum codec_state state;
 	struct codec_config s_cfg; /**< setup config */
 	struct codec_config r_cfg; /**< runtime config */
+	struct codec_interface *call;
 };
```

Comment:
The codec_interface field name should be called ops instead of call. As we traditionally call this kind of operations in SOF.

Reply:
Just a matter of taste I am fine with both proposals.

Reply:
@mrajwa if you don't mind please use ``ops``.

Reply:
ok, done.